src/Styleguide/Components/Footer.tsx

```diff
@@ -1,27 +1,49 @@
 import { Sans, Serif } from "@artsy/palette"
 import React from "react"
+import { GlobalState } from "Router/state"
 import styled from "styled-components"
 import { FlexDirectionProps } from "styled-system"
 import { Flex } from "Styleguide/Elements/Flex"
 import { Responsive } from "Styleguide/Utils/Responsive"
+import { Subscribe } from "unstated"
 
-export class Footer extends React.Component {
-  render() {
-    return (
-      <Responsive>
-        {({ xs }) => {
-          if (xs) return <SmallFooter />
-          else return <LargeFooter />
-        }}
-      </Responsive>
-    )
+interface Props {
+  mediator?: {
+    trigger: (action: string, config: object) => void
   }
 }
 
-export const LargeFooter = props => <FooterContainer flexDirection="row" />
-export const SmallFooter = props => <FooterContainer flexDirection="column" />
+export const Footer: React.SFC<Props> = props => {
+  return (
+    <Subscribe to={[GlobalState]}>
+      {({ state }) => {
+        return (
+          <Responsive>
+            {({ xs }) => {
+              if (xs)
+                return (
+                  <SmallFooter mediator={state.force && state.force.mediator} />
```

Comment:
Rather than guards could probably just use `defaultProps`, but can follow up in different pr